README.md

```diff
@@ -184,6 +185,24 @@ Also see our guides for [builds](./docs/builds.md) and [deployments](./docs/depl
 ### How the integration works
 When a workflow (e.g. GitHub Action) or development event (e.g. pull request, commit, branch) runs, the app receives a webhook from GitHub. The app then extract the issue key from the respective branch/commit/PR and send this information to Jira.
 
+## How the backfill works
+The app is designed to backfill historical data into Jira. Once you have installed and configured the app successfully, it will automatically trigger the backfilling process to update Jira with historical information such as pull requests, deployments, branches, builds, and commits.
+
+1. The backfilling process attempts to connect all branches that fulfill at least one of the following criteria:
+    - The branch name contains the issue key.
+    - The title of the latest pull request associated with the branch contains the issue key.
+    - The last commit message of the branch contains the issue key.
+2. All commits from the default branch will be backfilled. The commit message must contain the Jira issue key.
+3. Only the latest 50 commits from non-default branches will be backfilled.
+4. Unreachable commits (e.g. from deleted branches) will NOT be backfilled.
+5. All pull requests, regardless of their statuses, will be backfilled. The Jira issue key should be included either in 
```

Comment:
I thought the API only returns the currently opened PRs...

Reply:
I just connected an org and looks like it fetched MERGED PRs over 
<img width="500" alt="image" src="https://user-images.githubusercontent.com/20631664/220831416-5cbc9d50-fa6d-4cba-82e8-3e907cb679c2.png">
Also, according to the code, we are explicitly passing ALL states...
http://github.com/atlassian/github-for-jira/blob/edcad795ffea87447d305847e460abca3827ee42/src/sync/pull-request.ts#L68-L68

Please let me know if I'm mistaken...